packages/flutter/lib/src/services/system_channels.dart

```diff
@@ -27,6 +27,24 @@ class SystemChannels {
   ///  * [WidgetsBindingObserver.didPopRoute] and
   ///    [WidgetsBindingObserver.didPushRoute], which expose this channel's
   ///    methods.
+  ///
+  /// The following methods are used for the opposite direction data flow. The
+  /// framework notifies the engine about the route changes.
+  ///
+  ///  * `routePushed`, which is called when a route is pushed. (e.g. A modal
+  ///  is replaces entire screen.)
```

Comment:
"A modal is replaces entire screen" -> "A modal replaces the entire screen"

Reply:
Done.